src/main/java/org/intellij/sdk/codesync/alerts/ActivityAlerts.java

```diff
@@ -57,7 +58,14 @@ public static void remindLater() {
     Update the lock to skip the reminder for today and remind the user tomorrow.
      */
     public static void skipToday() {
+        Instant now = CommonUtils.getTodayInstant();
         Instant reminderInstant = CommonUtils.getTomorrowAlertInstant();
+
+        // if activity is shown before 4 AM then it was for yesterday's activity,
+        // and we need to show another notification after 4:30 PM today.
```

Comment:
1- 4PM
2- we need to show another notification after 4:30 PM today only if there has been an activity in past 24 h